packages/suite/src/components/wallet/Fees/CustomFee.tsx

```diff
@@ -36,16 +47,21 @@ interface CustomFeeProps<TFieldValues extends FormState> {
     control: Control;
     setValue: UseFormSetValue<TFieldValues>;
     getValues: UseFormGetValues<TFieldValues>;
-    changeFeeLimit?: (value: string) => void;
     composedFeePerByte: string;
 }
 
+// TODO: revisit with priority fees
```

Comment:
This logic probably works good for eth network but I am not sure if it makes sense for others. So it should be revisited